apps/src/applab/designMode.js

```diff
@@ -34,6 +34,11 @@ designMode.onDivApplabClick = function (event) {
   if (element.id === 'divApplab') {
     designMode.clearProperties();
   } else {
+    if ($(element).is('.ui-resizable')) {
+      element = element.children[0];
```

Comment:
We also make the "first child" assumption down in `makeUndraggable` -- maybe good to combine both into a function?
```javascript
var elm = $(':first-child', wrapper);
```